tests/test_utils.py

```diff
@@ -93,16 +95,20 @@ def test_client_authenticated(app):
 
 
 def test_create_test_user(app):
-    """Test for testutils.py:create_test_user().
+    """Tests for testutils.py:create_test_user().
 
     Demonstrates basic usage and context requirements."""
     ext = InvenioAccounts(app)
     email = 'test@test.org'
     password = '1234'
 
-    # Can't access the datastore outside of an application context
-    with pytest.raises(RuntimeError):
-        testutils.create_test_user(email, password)
+    # If pytest-flask isn't available we can't access the datastore outside of
```

Comment:
since we `pytest-flask` is part of test dependencies i'm not sure why do we need to keep it here.